posix.mak

```diff
@@ -375,6 +377,16 @@ unittest/%.run : $(ROOT)/unittest/test_runner
 %.test : $(LIB)
 	$(MAKE) -f $(MAKEFILE) $(addsuffix .test,$(patsubst %.d,%,$(wildcard $*/*)))
 
+%.debug_with_debugger : %.d $(LIB)
+	$(DMD) $(DFLAGS) -main -unittest $(LIB) -defaultlib= -debuglib= $(LINKDL) $<
```

Comment:
Oh right, Phobos the library is built without unit tests, then we want to build that one module with unit tests and use that instead of the module in Phobos.